src/Facebook/FileUpload/FacebookTransferChunk.php

```diff
@@ -106,7 +106,7 @@ public function getUploadSessionId()
     /**
      * Check whether is the last chunk
      *
-     * @return bool
+     * @return boolean
```

Comment:
I'm with @yguedidi on this one - `bool` is the type declaration keyword in PHP 7, so let's go with that one. :)